nilearn/input_data/tests/test_masker_validation.py

```diff
@@ -0,0 +1,65 @@
+import nibabel
+import numpy as np
+
+from sklearn.externals.joblib import Memory
+
+from nilearn.input_data.masker_validation import check_embedded_nifti_masker
+from nilearn.input_data import MultiNiftiMasker, NiftiMasker
+
+
+class C:
```

Comment:
You should always inherit from object.

Also, please use a slightly more explicit name, like "TestClass".